pandas_datareader/av/time_series.py

```diff
@@ -13,11 +14,10 @@ class AVTimeSeriesReader(AlphaVantage):
     ----------
     symbols : string
         Single stock symbol (ticker)
-    start : string, (defaults to '1/1/2010')
-        Starting date, timestamp. Parses many different kind of date
-        representations (e.g., 'JAN-01-2010', '1/1/10', 'Jan, 1, 1980')
-    end : string, (defaults to today)
-        Ending date, timestamp. Same format as starting date.
+    start : pd.Timestamp
```

Comment:
These need to match _BaseReader, which is ultimately at the end of a super chain. `str` should always be accepted even if `TimeStamp` is used internally rather than `datetime.datetime`.

Reply:
Many existing methods don't seem to follow:

https://github.com/pydata/pandas-datareader/blob/master/pandas_datareader/tsp.py#L54
https://github.com/pydata/pandas-datareader/blob/master/pandas_datareader/iex/daily.py#L95
https://github.com/pydata/pandas-datareader/blob/master/pandas_datareader/eurostat.py#L23

I'm not sure what would be a good update here ...

Reply:
@bashtage  @addisonlynch how would you like to see this moving forward?